src/components/navbar/Navbar.tsx

```diff
@@ -50,10 +42,17 @@ function Navbar() {
     // eslint-disable-next-line
   }, [role, authToken]);
 
-  const socket = getSocket();
-  socket.on("first-message", () => {
-    setCountNotifications(countNotifications + 1);
-  });
+  useEffect(() => {
+    setTotal(() => {
+      if (
+        (countNotifications && countNotifications.message) ||
+        (countNotifications && countNotifications.other)
```

Comment:
`countNotifications?.message || countNotifications?.other ` will in not work?

Reply:
fixed